ledger-service/http-json/src/test/scala/com/digitalasset/http/CliSpec.scala

```diff
@@ -271,8 +276,7 @@ final class CliSpec extends AnyFreeSpec with Matchers {
       )
     }
 
-    // TEST_EVIDENCE: Authentication: TLS configuration is parsed correctly from the config file
-    "should successfully load a complete config file" in {
+    "should successfully load a complete config file" taggedAs authenticationSecurity in {
```

Comment:
I prefer the previous test evidence description `TLS configuration is parsed correctly from the config file`. Could you please take that?